src/extensions/default/BrambleUrlCodeHints/camera/interface.js

```diff
@@ -101,6 +117,98 @@ define(function (require, exports, module) {
         // Update the photo component with the snapped photo
         this.photo.update();
 
+        // pattern from http://codepen.io/SitePoint/full/LVpNjp/
+        // $(function() {
+
+          //var Caman = require('caman').Caman;
+          var canvas = document.getElementById("canvas");  // canvas reference here
+          var ctx = canvas.getContext('2d');
+
+          /* Enable Cross Origin Image Editing */
+          var img = new Image();
+          img.crossOrigin = '';
+          img.src = self.photo.data;
+          console.log("img width: "+ img.width + " height " + img.height + "source: " + img.src);
+          console.log("save path in camera is: " + self.camera.savePath);
+
+          img.onload = function() {
+            canvas.width = img.width;
+            canvas.height = img.height;
+            console.log("loaded width: " + img.width + " height " + img.height);
+            ctx.drawImage(img, 0, 0, img.width, img.height);
+          }; // added missing semi-colon
+
+          var filtered = false;
```

Comment:
I think that's possible. In that case I will probably get rid of the variable "filtered" altogether. It's really only used [here](https://github.com/timmoy/brackets/blob/fdd141b6773f4ced276317f650aa81fc03f3812a/src/extensions/default/BrambleUrlCodeHints/camera/interface.js#L94) and has no bearing if we're saving through Caman regardless of if we apply a filter.